manual/googlechrome-extensions/googlechrome-editthiscookie/googlechrome-editthiscookie.nuspec

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>googlechrome-editthiscookie</id>
+    <version>1.0.0</version>
+    <title>EditThisCookie</title>
+    <authors>Francesco Capano</authors>
+    <owners>gep13 chocolatey</owners>
```

Comment:
Owners should be a comma separated list as per [.nuspec Reference](https://docs.nuget.org/ndocs/schema/nuspec)